src/Land/Model/LandModel.jl

```diff
@@ -0,0 +1,300 @@
+    @vars begin
+        soil::vars_state_auxiliary(land.soil, FT)
+    end
+end
+
+"""
+    vars_state_gradient(land::LandModel, FT)
+
+Names of the gradients of functions of the conservative state variables. Used to represent values before **and** after differentiation
+"""
+function vars_state_gradient(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_gradient(land.soil, FT)
+    end
+end
+
+"""
+    vars_state_gradient_flux(land::LandModel, FT)
+
+Names of the gradient fluxes necessary to impose Neumann boundary conditions
+"""
+function vars_state_gradient_flux(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_gradient_flux(land.soil, FT)
+    end
+end
+
+@doc """
+    flux_first_order!(
+        Land::LandModel,
+        flux::Grad,
+        state::Vars,
+        aux::Vars,
+        t::Real
+    )
+
+Computes and assembles non-diffusive fluxes in the model equations.
+"""
+
+function flux_first_order!(
+    land::LandModel,
+    flux::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+) end
+
+
+"""
+    function compute_gradient_argument!(
+        land::LandModel,
+        transform::Vars,
+        state::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify how to compute the arguments to the gradients.
+"""
+
+function compute_gradient_argument!(
+    land::LandModel,
+    transform::Vars,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+
+    compute_gradient_argument!(land, land.soil, transform, state, aux, t)
+end
+
+"""
+    function compute_gradient_flux!(
+        land::LandModel,
+        diffusive::Vars,
+        ∇transform::Grad,
+        state::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify how to compute gradient fluxes.
+"""
+
+function compute_gradient_flux!(
+    land::LandModel,
+    diffusive::Vars,
+    ∇transform::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+
+    compute_gradient_flux!(
+        land,
+        land.soil,
+        diffusive,
+        ∇transform,
+        state,
+        aux,
+        t,
+    )
+
+end
+
+"""
+    function flux_second_order!(
+        land::LandModel,
+        flux::Grad,
+        state::Vars,
+        diffusive::Vars,
+        hyperdiffusive::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify F_{second_order} for each conservative state variable
+"""
+
+function flux_second_order!(
+    land::LandModel,
+    flux::Grad,
+    state::Vars,
+    diffusive::Vars,
+    hyperdiffusive::Vars,
+    aux::Vars,
+    t::Real,
+)
+    flux_second_order!(
+        land,
+        land.soil,
+        flux,
+        state,
+        diffusive,
+        hyperdiffusive,
+        aux,
+        t,
+    )
+
+end
+
+"""
+    function update_auxiliary_state!(
+        dg::DGModel,
+        land::LandModel,
+        Q::MPIStateArray,
+        t::Real,
+        elems::UnitRange,
+    )
+
+Perform any updates to the auxiliary variables needed at the beginning of each time-step.
+"""
+
```

Comment:
```suggestion
```

Reply:
There's a bunch of these (we cannot have spaces between docstrings and structs/functions), just FYI

Reply:
Thank you! I guess we only fixed the ones where we exported the struct/function, and were getting errors with them in the documentation check. will make sure this is fixed in the future.